.editorconfig

```
@@ -0,0 +1,6 @@
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
+insert_final_newline = true
```

Please add trailing newlines where it was not automatically placed by your editor.

```suggestion
insert_final_newline = true

```